### PR TITLE
Add a code of conduct for the forecast hub

### PR DESCRIPTION
**Pull request by bsweger**
A code of conduct is required for participation in Netlify's open source program. This is the same one we discussed and adopted for the Hubverse. The only difference is the reporting contact list.

### CODE REVIEW

File: .github/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to any of the following community leaders responsible for enforcement: 
+Nicholas Reich (nick@umass.edu), Evan Ray (elray@umass.edu), or Becky Sweger
```
Comment (bsweger) [inline .github/CODE_OF_CONDUCT.md:63]: @nickreich @elray1 despite talking this through with Nick yesterday, I realized it didn't feel right to add a code of conduct without a way to report violations. So I put the three of us. Let me know if you'd prefer not to be listed.

**Review by nickreich — APPROVED** [review]
I am fine being listed. Thanks for adding this.

**Comment by nickreich** [comment]
@elray1 if you are ok being listed can you merge?

**Comment by elray1** [comment]
done

**Pull request merged**